pkg/reconciler/buildrun/resources/runtime_image.go

```diff
@@ -67,6 +67,13 @@ ENTRYPOINT [ {{ renderEntrypoint .Spec.Runtime.Entrypoint }} ]
 
 	// defultShellImage default image for a simple shell instance.
 	defultShellImage = "busybox:latest"
+
+	resultShellScript = `
+# Store the image digest
+cat /kaniko/shp-runtime-image-layout/index.json | grep digest | sed -E 's/.*sha256([^"]*).*/sha256\1/' | tr -d '\n' > "$(results.shp-image-digest.path)"
```

Comment:
this looks very human error-prone as it might lead to multiple commits to address issues. I do understand why this need to be here. I added in the next community meeting the topic on this feature, to see if it can be deprecated.